exaudfclient/base/exaudflib/exaudflib.cc

```diff
@@ -178,1562 +70,57 @@ void delete_vm(SWIGVM*& vm){
     }catch(...){
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1114","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): ");
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1114","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): ");
     }
     try{
         send_close(socket, msg);
         ::sleep(1); // give me a chance to die with my parent process
-    }  catch (SWIGVM::exception &err) {
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1115","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
+    }  catch (SWIGVMContainers::SWIGVM::exception &err) {
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1115","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
     }catch(std::exception& err){
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1116","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1116","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
     }catch(...){
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1117","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << ")");
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1117","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << ")");
     }
 
     try{
         stop_all(socket);
-    }  catch (SWIGVM::exception &err) {
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1118","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
+    }  catch (SWIGVMContainers::SWIGVM::exception &err) {
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1118","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
     }catch(std::exception& err){
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1119","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1119","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "): " << err.what());
     }catch(...){
-        PRINT_ERROR_MESSAGE(cerr,"F-UDF-CL-LIB-1120","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "):");
+        PRINT_ERROR_MESSAGE(std::cerr,"F-UDF-CL-LIB-1120","### SWIGVM crashing with name '" << socket_name << " (" << ::getppid() << ',' << ::getpid() << "):");
     }
     return 1;
 }
 
-
-} // namespace SWIGVMContainers
-
-
 extern "C" {
```

Comment:
Extract create_* into own cc file, maybe in impl

Reply:
Done